src/sankoff_handler.hpp

```diff
@@ -129,7 +133,7 @@ TEST_CASE("SankoffHandler: Tests on single site sequence.") {
       1., 2.5, 0., 2.5,       //
       2.5, 1., 2.5, 0.;       //
 
-  SankoffHandler sh = SankoffHandler(costs, site_pattern);
+  SankoffHandler sh = SankoffHandler(costs, site_pattern, "_ignore_mmaped_psv.data");
```

Comment:
Was this supposed to be `_ignore/mmapped_psv.data`?

Reply:
resolved